blockly-ws-search/src/WorkspaceSearch.js

```diff
@@ -42,7 +42,7 @@ export class WorkspaceSearch {
 
     /**
      * A list of blocks that came up in the search
-     * @type {!Array.<Blockly.Block>}
+     * @type {!Array.<Blockly.BlockSvg>}
```

Comment:
Why the change to BlockSvg?

Reply:
Keeping track Block vs BlockSvg started to seem like too much overhead, especially after introducing a way for users to custom set the search function (and we are expecting the blocks returned to be BlockSvg). Since we take in a WorkspaceSvg, we're only sort of supporting non-rendered workspaces. We should probably talk more offline.